terra-jupyter-r/Dockerfile

```diff
@@ -38,8 +38,8 @@ RUN apt-get update \
 	libgit2-dev \
 	libxml2-dev \
 	libgfortran-7-dev \
-	r-base-dev \
-	r-base-core \
+	r-base-dev=4.1.3-1.1804.0 \
+	r-base-core=4.1.3-1.1804.0 \
```

Comment:
we had to pin the R version to 4.1 until bioconductor is ready to release bioconductor 3.15 with r 4.2